lib/IDE/CommentConversion.cpp

```diff
@@ -538,17 +544,25 @@ class DoxygenConverter : public MarkupASTVisitor<DoxygenConverter> {
 
 public:
   DoxygenConverter(llvm::raw_ostream &OS, unsigned Indent = 0)
-    : OS(OS), Indent(Indent), IsFreshLine(true) {
+    : OS(OS), Indent(Indent), IsFreshLine(true), IsEmptyComment(true) {
     printIndent();
```

Comment:
Now that you mention it, I don't know if the `Indent` parameter is important. I don't see any construction of `DoxygenConverter` in the codebase that uses a custom `Indent` argument.